### DISCUSSION TITLE
nightscout not working via google cloud

### DISCUSSION THREAD

**Discussion by annanicolaou23** [NightscoutFoundation/xDrip#2976] [General]
SInce yesterday nightscout is not working for me. I use Google Cloud. I did not change anything. When I go to my domain is does not exist. Here is a screenshot  of the error message I get https://photos.app.goo.gl/RoPPVRBmbYthJydY8

I tried the fix suggested in a previous post (https://github.com/NightscoutFoundation/xDrip/discussions/2788?fbclid=IwAR0uqg4Qraron7DpRPLfKwqlpmu-oQb5zw_EDzQyQ28twDTHL_tLBwDGmd4#discussioncomment-5659798) but no luck. 

Any help greatly appreciated!

**Comment by Navid200** [comment]
Please look at this thread where I have explained how you can attach an image.
https://github.com/NightscoutFoundation/xDrip/discussions/1900

Then , post the image so that I can see it.
But, more importantly, I need to see a screenshot of your status page.

**Reply by annanicolaou23** [reply]
![image](https://github.com/NightscoutFoundation/xDrip/assets/141227612/df212f2d-3047-4630-82f3-dd8051e3001b)
![Screenshot_20230802-121657_xDrip+~2](https://github.com/NightscoutFoundation/xDrip/assets/141227612/e57b861e-cf93-4657-bebb-e462a7b7d388)

**Reply by annanicolaou23** [reply]
thanks Navid for any help, I have no idea how to fix this.

**Comment by Navid200** [comment]
This is what I mean by the status page.
https://navid200.github.io/xDrip/docs/Nightscout/Status.html

**Comment by Navid200** [comment]
We have made a few updates since you first installed.
The benefit of those updates that they make it easier to troubleshoot.
The updates are all incremental.  So, there is no need to reinstall.

When you have time, please follow this page.
If at any point, you feel that you prefer to just do a fresh install, please just take a break.  There is no need.
And please don't hesitate asking for confirmation.

This is the update procedure.

**Comment by Navid200** [comment]
https://navid200.github.io/xDrip/docs/Nightscout/NS_SyncExecutables.html

**Reply by annanicolaou23** [reply]
many thanks Navid, I will try the update procedure.

**Reply by annanicolaou23** [reply]
![image](https://github.com/NightscoutFoundation/xDrip/assets/141227612/30fd6bf6-3d01-4086-a1d9-d04ad20fe11f)

**Reply by annanicolaou23** [reply]
Hi Navid, I did all the updates as you suggested but there still seems to be an issue with the FreeDNS name and IP. It says Mismatch. Could this be cause I am in a different country? When I type in my cloudwatch.net domain it does not exist. Any ideas? Many thanks!

**Reply by Navid200** [reply]
The mismatch flag occasionally show up when it is not real.  
Unless it is real, if you leave the status page (Return option) and come back to the status page, it will be gone.
Do you see it consistently?

**Reply by Navid200** [reply]
Can you not see your Nightscout in a browser?

**Reply by annanicolaou23** [reply]
its still there
![image](https://github.com/NightscoutFoundation/xDrip/assets/141227612/e5b693d3-1c6d-4ea8-af69-6128a8bad4e5)

**Reply by annanicolaou23** [reply]
No, I cannot not see the nightscout in the browser, and when I go to the FreeDNS page it stays this domain is broken since 2 Aug...Do i need to make a new url with a different domain?

**Reply by Navid200** [reply]
Can I see a screenshot of the message that says the domain is broken?  Please before posting the image, blackout the hostname in it.

**Reply by Navid200** [reply]
I'd like to know exactly what happened.
It's possible that you just need to reactivate your account.  But, it may also be possible that someone has removed a domain.  I have never seen that.

Based on your description, you can create a new subdomain and then, follow this to change your hostname to the new one:
https://navid200.github.io/xDrip/docs/Nightscout/ChangeHostname.html

I would just like to see this so that if needed, improve the guide to help others who may experience this.

**Reply by annanicolaou23** [reply]
![Screenshot_20230806-191113_Chrome~2](https://github.com/NightscoutFoundation/xDrip/assets/141227612/328119c6-bdcd-468b-920c-2b30b646616e)

**Reply by Navid200** [reply]
I see, the whole domain.  It even says to contact the owner.
So, your only option is to create a new subdomain on a different domain.
After you have create a new subdomain, you can switch your hostname to it as I explained in the link I have provided.
After you do that, everything will work and your database will remain where it was and the data that is currently in the queue will all be uploaded to your Nightscaut again.

Thanks a lot for reporting this and proving screenshots.

Please let me know if anything doesn't work.

**Reply by annanicolaou23** [reply]
![Screenshot_20230806-195037_Chrome](https://github.com/NightscoutFoundation/xDrip/assets/141227612/9b5a55dd-e8f4-47e2-87c0-d5686c8d4fc5)

**Comment by annanicolaou23** [comment]
Many thanks Navid, I did contact the owner but have not heard back. I was
afraid to create a new subdomain in case it messed things up, but will try
to do it following your link and let you know how it goes.

**Reply by Navid200** [reply]
No, there will be no problem.
You can have 5 different subdomains for free.  And each can be on a different domain, or some can be on the same domain.  There is no restriction.

**Reply by annanicolaou23** [reply]
Thanks Navid, I created a new domain as you suggested and nightscout is up and running but I can't seem to connect it to xdrip. @
![Screenshot_20230807-133156_xDrip+~2](https://github.com/NightscoutFoundation/xDrip/assets/141227612/670c919a-9de8-4a5a-95a2-9cfaa7842b4f)

**Reply by annanicolaou23** [reply]
How do I do that? Thanks again!

**Reply by Navid200** [reply]
This page has a lot about that to help:
https://navid200.github.io/xDrip/docs/Nightscout/NS_setup.html

You can see the syntax on the page.
You can see a video clip I show the entire procedure for setting up xDrip using a QR code.

I don't know if you have seen that or not.

**Comment by annanicolaou23** [comment]
Thank you so much for your help again Navid. It all works fine now with the
new domain. You are a lifesaver!

**Reply by Navid200** [reply]
Awesome!
You're welcome.
